agent/agent.go

```diff
@@ -447,6 +535,17 @@ func stopServiceInputs(inputs []*models.RunningInput) {
 	}
 }
 
+func (a *Agent) StopInputPlugin(input string) {
+	a.inputStatus[input] = "STOP"
```

Comment:
Why do you do this to stop a plugin? Seems a little hacky. Why not just stop the accumulator first, and then stop the plugin?

Reply:
That was for demo purposes but we now are using a map of channels!